staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -171,6 +171,7 @@ func ListResource(r rest.Lister, rw rest.Watcher, scope *RequestScope, forceWatc
 		ctx := req.Context()
 		// For performance tracking purposes.
 		ctx, span := tracing.Start(ctx, "List", traceFields(req)...)
+		req = req.WithContext(ctx)
```

Comment:
we don't have this for any other verb. Is it only needed for watch? From the PR description it does not look like. Why don't we add that everywhere to get proper serialization spans?

Reply:
@sttts sorry, I forgot to add it for other verbs. updated.